com.github.fries1234.ncsa-mosaic.appdata.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>com.github.fries1234.ncsa-mosaic</id>
+    <name>NCSA Mosaic</name>
+    <summary>The NSCA Mosaic Web Browser</summary>
+    <metadata_license>CC-BY-SA-3.0</metadata_license>
+    <project_license>LicenseRef-proprietary=https://github.com/fries1234/ncsa-mosaic/blob/master/COPYRIGHT</project_license>
```

Comment:
should the project license be "NCSA" ?

Reply:
re-reading the license, it might not even be open source, and therefore there could be legal issues.

IANAL.

Reply:
I misread the license. Only libdtm is under the Public Domain which is not compiled into the browser build. I am okay with marking the browser as proprietary.